backend/apps/core/validators.py

```
@@ -0,0 +1,89 @@
+from django.http import JsonResponse
```

Ruff (0.8.2)

Missing docstring in public module

(D100)